.gitignore

```diff
@@ -99,7 +99,8 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+.install.stamp
+poetry.lock
```

Comment:
Q: what's this... I think we're gitignoring poetry.loc, but also checking it in with this PR?

Reply:
Good catch Peter TY! I did not mean to un-comment that!